config/initializers/theme_loader.rb

```diff
@@ -3,15 +3,34 @@
 # It is used by our config/routes.rb to decide which route extension files to load.
 $alaveteli_route_extensions = []
 
+def theme_root(theme_name)
+  Rails.root.join('lib/themes', theme_name)
+end
+
 def require_theme(theme_name)
-  theme_lib = Rails.root.join 'lib', 'themes', theme_name, 'lib'
+  root = theme_root(theme_name)
+  theme_lib = root.join('lib')
   $LOAD_PATH.unshift theme_lib.to_s
-  theme_main_include = Rails.root.join theme_lib, "alavetelitheme.rb"
-  if File.exist? theme_main_include
-    require theme_main_include
-  end
+
+  theme_main_include = theme_lib.join('alavetelitheme.rb')
+
+  return unless File.exist?(theme_main_include)
+
+  require theme_main_include
+
+  Rails.configuration.paths.add(
+    'config/refusal_advice',
+     with: root.join('config/refusal_advice'),
+     glob: '*.yml'
+  )
 end
 
+Rails.configuration.paths.add(
```

Comment:
Did I mean to commit this? Hmm.

Reply:
Probably not – it's unlikely we'll provide any default refusal advice configuration.

Reply:
We can drop this; no need to worry about none theme as it'll only be us using it.